psat_server_web/atlas/accounts/migrations/0001_initial.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.1.13 on 2024-10-29 11:23
```

Comment:
This file should not be in the pull request. Can we remove it and the entire migrations directory. No migrations should ever be checked in. I suspect this was added by accident!

Reply:
Happy to remove but not done by accident, committing migrations allows the whole database to be reproduced in a dev environment easily and (I think) is standard practice. Django wrote them to be version controlled and sometimes we will have to write manual migrations, which will definitely need to be checked in! We can discuss this afternoon of course :)